config.tfvars

```diff
@@ -77,6 +77,16 @@ jira_db_instance_class       = "db.t3.micro"
 jira_db_allocated_storage    = 100
 jira_db_iops                 = 1000
 
+# If you want to restore the database from a snapshot, uncomment the following line and provide the snapshot identifier. This will
+# restore the database from the snapshot and will not create a new database.
+#jira_db_snapshot_identifier = "<SNAPSHOT_IDENTIFIER>"
+
+# (Optional) The master username for the database instance. If you provided, the username is "postgres".
+#jira_rds_master_username     = "<RDS_MASTER_USERNAME>"
```

Comment:
```suggestion
#jira_ds_master_username     = "<DB_MASTER_USERNAME>"
```

Reply:
Or if we want to use the `rds` prefix, then I think we should also use `jira_rds_snapshot_identifier`. I do prefer the `db` prefix because it is less prescriptive.